addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -1,6 +1,6 @@
 class ACEGVAR(medical_treatment,actions) {
     class BasicBandage {
-        treatmentTime = QUOTE(call ACEFUNC(medical_treatment,getBandageTime) / 2);
+        treatmentTime = QUOTE(call ACEFUNC(medical_treatment,getBandageTime));
```

Comment:
Could be fully removed or commented out but w/e.

Reply:
I thought about removing it but now I think it will not stay like this for a long time. It will need some adjustments, for now I just don't know how to do this properly.